### PR TITLE
additions to network topolgy 

### PR DESCRIPTION
**Pull request by IC-encipher**
added graphs and explanations of graphs to better explain what a neighbourhood looks like and examples of what clients would be connected to if a neighbourhood is not properly formed.

### CODE REVIEW

**Comment by xvk-64** [comment]
This is a lot of text! I agree that additional examples would be useful for understanding, but maybe put them in an additional file?

**Comment by IC-encipher** [comment]
will do

**Comment by IC-encipher** [comment]
would it be worth leaving one example of a neighbourhood in the main spec document for example purposes also or should I move all the graphics?

**Comment by IC-encipher** [comment]
have moved graphics to new file

**Comment by IC-encipher** [comment]
should be visible in the commits now

File: readme.md [hunk 1]
```diff
@@ -30,9 +30,10 @@ Server (Owner of the receiver)
 Client (Receiver)
 ```
 
-If a server "owns" a client, that just means that the client is connected to that server. (Since clients only connect to one server at a time)
+If a server "owns" a client, that just means that the client is connected to that server (Since clients only connect to one server at a time). The transport layer of this protocol uses Websockets (RFC 6455). 
 
-The transport layer of this protocol uses Websockets (RFC 6455)
+You can call the server you are connected to your homeserver. You can only connect to users connected to your home server or users on servers directly connected your home server. Refer to the additional file network topology examples, to see examples of neighbourhoods and other potential network arrangments. 
+To illustrate this consider these the diagrams below:
```
Comment (xvk-64) [inline readme.md]: Forgot to remove this

**Comment by IC-encipher** [comment]
removed 
To illustrate this consider these the diagrams below:

**Comment by xvk-64** [comment]
OK

**Pull request merged**